workshops/image_editor/README.md

```diff
@@ -0,0 +1,457 @@
+    </div>
+  </div>
+
+    <!--Section #3-->
+  <div class="row">
+    <div class="col-md-4">
+    </div>
+    <div class="col-md-4">
+    </div>
+    <div class="col-md-4">
+    </div>
+  </div>
+  
+</div>
+```
+Each ```<div>``` tag simply creates a container where the rows/columns will be in. What we actually need to understand are the classes that we are using:
+
+- The main container has the class **container**-**fluid**, which is like any other div but that can expand/shrink depending on how big your browser is displaying the website. This is what's known as [responsive web design](https://www.w3schools.com/html/html_responsive.asp).
+- Column classes indicate the number of columns you’d like to use out of the possible 12 per row. We need three equal-width columns across, so we can use  ```col-md-4```. We also need a single column for the first row, which is why we are using ```col-md-12```.
+
+## Section 1
+In this section we are going to put an input where people will be able to choose what image they want to edit. To do this we are going to need a form, a text input and a button with these values:
+```html
+<!--Section #1-->
+<div class="row">
+  <div class="col-md-12 text-center mb-3 mt-3">
+    <form id="imgurl">
+      <input type="text" placeholder="Image URL" id="onlineurl">
+      <button type="submit">Change Image</button>
+    </form>
+  </div>
+</div>
+```
+Let's break this down:
+- The class ```text-center``` keeps the input and the button elements in the center of the column.
+- The class ```mb-3``` gives the column 3px of margin to its bottom.
+- The class ```mt-3``` gives the column 3px of margin to its top.
+
+<details><summary>Here's how our HTML document looks so far:</summary>  
+  
+```html
+<div class="container-fluid">
+
+  <!--Section #1-->
+  <div class="row">
+    <div class="col-md-12 text-center mb-3 mt-3">
+      <form id="imgurl">
+        <input type="text" placeholder="Image URL" id="onlineurl">
+        <button type="submit">Change Image</button>
+      </form>
+    </div>
+  </div>
+  
+  <!--Section #2-->
+  <div class="row">
+    <div class="col-md-4">
+    </div>
+    <div class="col-md-4">
+    </div>
+    <div class="col-md-4">
+    </div>
+  </div>
+
+    <!--Section #3-->
+  <div class="row">
+    <div class="col-md-4">
+    </div>
+    <div class="col-md-4">
+    </div>
+    <div class="col-md-4">
+    </div>
+  </div>
+  
+</div>
+```
+</details>
+
+![Section 1 Result](https://cloud-cwwxoedf8.vercel.app/0section1.png)
+
+## Section 2
+In section 2 we'll just have to add an image tag inside of the **second column** of the **second row**:
+```html
+<!--Section #2-->
+<div class="row">
+  <div class="col-md-4">
+  </div>
+  <div class="col-md-4 text-center">
+    <img src="https://assets.hackclub.com/flag-orpheus-top.svg" id="targetimage" alt="">
+  </div>
+  <div class="col-md-4">
+  </div>
+</div>
+```
+- The class ```text-center``` as seen before, keeps all the elements in the center.
```

Comment:
text-center will only center the text right? and not ALL the elements. So I think that's a bit wrong to say "all the elements to center"

Reply:
No, it actually centers the elements.